spyderlib/plugins/shortcuts.py

```diff
@@ -159,17 +63,12 @@ def data(self, index, role=Qt.DisplayRole):
         column = index.column()
         if role == Qt.DisplayRole:
             if column == CONTEXT:
-                return to_qvariant(shortcut.context)
+                return to_qvariant(_(shortcut.context.capitalize()))

```

Comment:
Not really... the gettext scripts when ran, go through the source code and grab any text and create the .po files. This process is done offline every time we need to update the translations. On runtime the _() will fetch the corresponding translated text...